docs/spec/protocol.rst

```diff
@@ -74,10 +73,9 @@ be used in every context where normal types can::
   close_all([f, r])  # OK!
   close_all([1])     # Error: 'int' has no 'close' method
 
-Note that both the user-defined class ``Resource`` and the built-in
-``IO`` type (the return type of ``open()``) are considered subtypes of
-``SupportsClose``, because they provide a ``close()`` method with
-a compatible type signature.
+Note that both the user-defined class ``Resource`` and the built-in ``IO`` type
+(the return type of ``open()``) are assignable to ``SupportsClose``, because
```

Comment:
Not relevant to this PR's purpose -- but lol, if only `IO` were the [return type of `open()`](https://github.com/python/typeshed/blob/e285e52e1385a3c63a850dd5a2e8b2c744d7293a/stdlib/builtins.pyi#L1527-L1612).